apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node.scss

```
@@ -0,0 +1,106 @@
+@import 'colors';
+@import '../../../styles/global/variables';
+
+:host {
+  max-width: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+:host ::ng-deep button {
+  font-size: 14px;
+  outline: none;
+}
+
+:host ::ng-deep .ba-uxd-node-content p {
+  margin-bottom: 0;
+}
+
+.ba-uxd-node {
+  margin-bottom: 68px;
+  padding-left: 24px;
+  border-left: 2px solid #ccc;
+}
+
+.ba-uxd-node-wrapper {
+  width: 100%;
+}
+
+@media only screen and (min-width: $screen-lg) {
+  .ba-uxd-node-wrapper {
+    width: 76%;
+    margin: 0 auto;
+  }
+}
+
+.ba-uxd-node-content {
+  width: fit-content;
+  margin-bottom: 20px;
+}
+
+.ba-uxd-edge-button {
+  height: 32px;
+  min-width: 40px;
+  padding-left: 15px;
+  padding-right: 15px;
+  border-radius: 3px;
+  border: none;
+  color: white;
+  background-color: $royalblue-600;
+
+  &:hover {
+    background-color: $royalblue-500;
+  }
+
+  &:disabled {
+    background-color: $gray-300;
+
+    &:hover {
+      background-color: $gray-300;
+    }
+  }
+}
+
+.ba-uxd-yes-edge {
+  height: 44px;
+  width: 44px;
+  border-radius: 22px;
```

Will make it round, no matter the width and height. If that were to change at some point.
```suggestion
  border-radius: 50%;
```